src/Acts/CamdramBundle/Service/ModerationManager.php

```diff
@@ -195,6 +188,7 @@ public function notifySocietyChanged($entity)
         }
     }
 
+    // FIXME properly...
```

Comment:
Could you possibly leave a better description of what exactly needs work here? (eg. methods that need refactoring or whatever!)

Reply:
Sorry, I try not to commit that sort of comment since I'm bound to forget what it meant! Eventually it came back to me:
* This method is called only in unreachable code, i.e. the event listener is broken
* If the method was called, it calls `getVenue()` on a show, which no longer exists.

Fixing it properly therefore means setting up the event listener again and making it send an appropriate notification email.